### DISCUSSION TITLE
怎样能够设置自己的反代服务器的地址

### DISCUSSION THREAD

**Discussion by ZYLHL** [spencerwooo/onedrive-vercel-index#747] [Q&A] [answered]
> **强调** 我说的反向代理不是指Cloudflare！不是Cloudflare！不是Cloudflare！

从国内访问国际版ofb大多数情况虽说都挺正常，但因为各方面因素速度一直一般般，有些省份地区甚至因为下载太慢时间久了直接提示下载超时。
通过nginx设置反向代理的方式让拥有优秀的出国和回国线路的境外VPS的中转ofb的流量达到从国内访问更快的作用。
我想问一下怎样能够设置自己的反代服务器的地址，让在网页上不论是点直链下载还是在线预览视频都默认通过自己设置的反向代理服务器去中转？

**Comment by ZYLHL** [comment]
我根据这篇文章已经配置好了反向代理服务器的设置，通过手动替换 `*-my.sharepoint.com` 为**反向代理域名**的方式测试中转成功，速度能跑满VPS总的100M带宽。
https://zhuanlan.zhihu.com/p/326043846

**Reply by ZYLHL** [reply]
问题主要是怎么设置到 ovi 中，通过点下载和点在线预览的时候程序**自动**将 `*-my.sharepoint.com` 替换成反向代理域名？💭

**Reply by NAILIANGCHI** [reply]
将反向代理地址修改在哪个目录？🤔

**Comment by Arstman** [comment]
直接部署在cloudflare上不就完事了

**Reply by pseudoxx** [reply]
Cloudflare不支持ssr功能的next.js，不能部署的。

**Comment by nmnmcc** [comment, marked as answer]
修改raw.js，使用split替换1drv。
